lib/rbnacl/init.rb

```diff
@@ -5,7 +5,7 @@ module RbNaCl
   # Defines the libsodium init function
   module Init
     extend FFI::Library
-    ffi_lib ["sodium", "libsodium.so.18", "libsodium.so.23", "libsodium.so.26"]
+    ffi_lib ["sodium", "libsodium.so.18", "libsodium.so.23", "libsodium.so.26", "libsodium-26"]
```

Comment:
It would be nice to reduce this duplication with RbNacl::Sodium class. Any suggestion on how to do it--define a shared constant?